api-authorization/server/lib/authorization-middleware.js

```diff
@@ -0,0 +1,32 @@
+import jwt from 'jsonwebtoken';
+import { ClientError } from './client-error.js';
+
+export function authMiddleware(req, res, next) {
+  try {
```

Comment:
This try/catch is very thorough. In error middleware, though, it's not necessary since if it throws, Express will automatically send it on to the next handler.